arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -545,6 +545,104 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/powerOff": {
```

Comment:
I am perfectly fine with having consistent naming for start/stop we chose this purely to be consistent with VMs.  If you don’t care about consistency, then I would also prefer consistency using start/stop.